components/Pages/ServiceUser/Packages/CarePackageStatus.js

```diff
@@ -33,7 +35,7 @@ export const CarePackageStatus = ({ status, packageData }) => {
   };
 
   const activeStatus = getStatus(status, packageData);
-  const color = activeStatus === 'Active' ? '#00664F' : getTagColorFromStatus(status);
+  const color = activeStatus === 'Active' ? 'green' : getTagColorFromStatus(status);
```

Comment:
Also fixed text of the status

Reply:
![image](https://user-images.githubusercontent.com/77606518/142846007-a1fd6e2e-9bf5-465b-92f1-cfa848efc06a.png)

service-user/920623e6-9e0c-499a-b08e-6a5e5484c2b8/packages